utils/ansible/ftest/templates/daos-make.sh.j2

```diff
@@ -13,13 +14,18 @@ SERVERS_LIST="{{ groups["daos_servers"] | join(',') }}"
 {% if "daos_clients" in groups and groups["daos_clients"] | length > 0 %}
 CLIENTS_LIST="{{ groups["daos_clients"] | join(',') }}"
 {% endif %}
+BUILD_TYPE="{{ daos_build_mode | default(daos_build_mode_default) }}"
 
 SCONS_EXE=scons-3
```

Comment:
Everything should be `scons` now, `scons-3` was about transitioning from python2 which should finally be complete.

Reply:
- [ ] Replace scons-3 with scons

Reply:
> * [x]  Replace scons-3 with scons

Fixed with commit 07b4777df